Library/Formula/fish-tank.rb

```diff
@@ -0,0 +1,11 @@
+class FishTank < Formula
+  homepage "https://github.com/terlar/fish-tank"
+  url "https://github.com/terlar/fish-tank/archive/0.1.1.tar.gz"
+  sha1 "b0be92bf01dae05e0efced363139d0d813365485"
+
+  head "https://github.com/terlar/fish-tank.git"
+
+  def install
+    system "make", "install"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!